src/testdrive/src/format/protobuf/nested.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+syntax = "proto3";
+
+enum Binary {
+  ZERO = 0;
+  ONE = 1;
+}
+
+message NestedOuter {
+  double double = 1;
+  float float = 2;
+  int32 int32 = 3;
+  int64 int64 = 4;
+  sint32 sint32 = 5;
+  sint64 sint64 = 6;
+  sfixed32 sfixed32 = 7;
+  sfixed64 sfixed64 = 8;
+  bool bool = 9;
+  string string = 10;
+  bytes bytes = 11;
+
+  Binary binary = 12;
+
+  repeated NestedInner nested = 13;
+}
+
+message NestedInner {
+  double double = 1;
+  float float = 2;
+  int32 int32 = 3;
+  int64 int64 = 4;
+  sint32 sint32 = 5;
+  sint64 sint64 = 6;
+  sfixed32 sfixed32 = 7;
+  sfixed64 sfixed64 = 8;
+  bool bool = 9;
+  string string = 10;
+
+  Binary binary = 11;
```

Comment:
Is it necessary for the test for the proto to have all these fields? It seems like it just makes it harder to debug and use.

Reply:
Probably not necessary! I just wanted to make sure all of the types worked.